apps/docs/app/layout.css

```diff
@@ -90,6 +90,12 @@ p {
     text-decoration: underline;
```

Comment:
![image](https://github.com/gsoft-inc/wl-hopper/assets/361632/d4313c4f-ee03-4d71-ba57-b8a2a450a630)
I think migration notes should be a list.